middleware/householdMiddleware.go

```diff
@@ -140,12 +139,12 @@ func (hm HouseholdMiddleware) CreateCalendar(w http.ResponseWriter, r *http.Requ
 	if userErr != nil {
 		json.NewEncoder(w).Encode(userErr.Error())
 	} else {
-		var calendar models.Calendar
-		_ = json.NewDecoder(r.Body).Decode(&calendar)
 		bearerToken := r.Header.Get("Authorization")
 		user, _ := hm.um.repository.GetUserByAccessToken(strings.ReplaceAll(bearerToken, "Bearer ", ""))
-		calendar.HouseholdID, _ = primitive.ObjectIDFromHex(user.HouseholdId)
-		payload, err := hm.controller.CreateCalendar(calendar, hm.calendarRepo)
+		// Should this be a separate model, something like CalendarCreateRequest that only has the StartDate?
```

Comment:
in a pure REST sense, yes, but I'm cool with this personally

Reply:
Yea at some point I'll go through and implement CQRS but there's other cleanup projects *side-eyes tests* that I need to do first.